/path_node:
  ros__parameters:
    coordenadas_x: [1.0, 1.0, 0.0, 0.0]
    coordenadas_y: [0.0, 1.0, 1.0, 0.0]
    lin_vel: [0.025, 0.380]
    ang_vel: [0.290, 3.000]
    area: 10.0